RUFAS/biophysical/manure/storage/daily_spread.py

```diff
@@ -0,0 +1,25 @@
+from RUFAS.biophysical.manure.storage.storage import Storage
+from RUFAS.current_day_conditions import CurrentDayConditions
+from RUFAS.data_structures.animal_to_manure_connection import ManureStream
+from RUFAS.rufas_time import RufasTime
+
+
+class DailySpread(Storage):
+    def __init__(
+        self,
+        name: str,
+        cover: str,
+        surface_area: float,
+    ):
+        """Initialize DailySpread object."""
+        super().__init__(
+            name=name,
+            is_housing_emissions_calculator=False,
+            cover=cover,
+            storage_time_period=1,
```

Comment:
```suggestion
            storage_time_period=storage_time_period,
```
(I don't know if this is actually necessary, I was just mimicking the init of another processor here)

Reply:
It works the same, but it's good to keep consistent.